packages/flutter/test/painting/decoration_test.dart

```diff
@@ -583,4 +583,15 @@ void main() {
       expect(call.positionalArguments[2].center, outputRect.center);
     }
   });
+
```

Comment:
You should also add a scale test similar to [this paintImage scale test](https://github.com/flutter/flutter/pull/54258/files#diff-42ee30850dae3103f2cfc1e4a715532aR420). The test in particular tests `paintImage`, but you can combine elements from [this BoxDecoration test](https://github.com/flutter/flutter/pull/54258/files#diff-42ee30850dae3103f2cfc1e4a715532aR223) with the scale test for `paintImage` to achieve the outcome you need.